internal/oci/runtime_oci.go

```diff
@@ -711,7 +711,6 @@ func (r *runtimeOCI) UpdateContainerStatus(c *Container) error {
 		}
 		out, err := cmd.Output()
 		if err != nil {
-			logrus.Errorf("Failed to update container state for %s: %v", c.id, err)
```

Comment:
Why is this line dropped?

Reply:
it's duplicated with error messages below

Reply:
I see